### ISSUE TITLE
LibreQoS.py - parameter for cleaning up all the queues

### ISSUE THREAD

**Issue by interduo** [rchac/LibreQoS#55]
I see inside the code function clearPriorSettings (https://github.com/rchac/LibreQoS/blob/main/v1.1/LibreQoS.py#L96).

For debugging purposes - we need to clean all the queues sometimes.
It would be marvelous when we could run `LibreQoS.py --clear-rules`

I also check that
`systemctl stop LibreQoS`
dont clean tc rules (that is fine) and don't touch XDP

**Comment by interduo** [comment]
When interface is down the XDP should u load because:
```
BPF filter is not automatically unloaded from the network device. Manual filter unload:
ip link set dev <interface> xdp off
```

**Comment by rchac** [comment]
Is there a way to call a python function to unload xdp when ```systemctl stop LibreQoS``` occurs?

**Comment by interduo** [comment]
write `LibreQoS.py - - clear-rules`
The add this to systemd service in `ExecStopPost=`

**Comment by rchac** [comment]
Added argument ```--clearrules```. https://github.com/rchac/LibreQoS/blob/LibreQoS-1.2-IPv6/v1.2/LibreQoS.py Once its merged, I will update wiki for v1.2 so  ```systemctl stop LibreQoS``` will do what is intended.

**Comment by interduo** [comment]
> Is there a way to call a python function to unload xdp when systemctl stop LibreQoS occurs?

shell('cmd') os.system('cmd' ) should be enough
